### ISSUE TITLE
ValidationError popup shows only first char of the error string

### ISSUE THREAD

**Issue by QuantumDeveloper** [MahApps/MahApps.Metro#1926]
I am using INotifyDataErrorInfo interface with MA ValidationErrorTemplate like this:
```
<Style x:Key="ComboBoxStyle" TargetType="ComboBox" BasedOn="{StaticResource {x:Type ComboBox}}">
         <Setter Property="VerticalAlignment" Value="Center"/>
         <Setter Property="VerticalContentAlignment" Value="Center"></Setter>
         <Setter Property="HorizontalAlignment" Value="Center"></Setter>
         <Setter Property="MinWidth" Value="50"></Setter>
         <Setter Property="Width" Value="150"></Setter>
         <Setter Property="Validation.ErrorTemplate"
                Value="{StaticResource ValidationErrorTemplate}" />
      </Style>
```

And use it like this:
```
<ComboBox Grid.Column="1" Grid.Row="4" Style="{DynamicResource ComboBoxStyle}" ItemsSource="{Binding Source={StaticResource FeatureLevelEnum}}" 
                   SelectedValue="{Binding Path=GraphicsSettingsViewModel.SelectedFeatureLevel, Mode=TwoWay, 
            UpdateSourceTrigger=PropertyChanged, ValidatesOnNotifyDataErrors=True, NotifyOnValidationError=True}"/>
```
And validating property:
```
private void ValidateFeatureLevel(FeatureLevel featureLevel)
      {
         const string propertyKey = "SelectedFeatureLevel";
         if (SelectedAdapterInfo.Adapter != null)
         {
            var isSupported = Device.IsSupportedFeatureLevel(SelectedAdapterInfo.Adapter, featureLevel);

            if (isSupported == false)
            {
               /* Update the collection in the dictionary returned by the GetErrors method */
               if (!validationErrors.ContainsKey(propertyKey))
               {
                  validationErrors.Add(propertyKey, "This feature level is not supported on current video adapter");
               }
            }
            else if (validationErrors.ContainsKey(propertyKey))
            {
               /* Remove all errors for this property */
               validationErrors.Remove(propertyKey);
            }
            RaisePropertyChangedByName("HasErrors");
            RaiseErrorsChanged(propertyKey);
         }
      }
```

But as A result I see only first char of validation string:
![image](https://cloud.githubusercontent.com/assets/6085009/7702107/51532430-fe36-11e4-8d9a-36159b251ab3.png)

Could you please help me and say why this happening?

**Comment by punker76** [comment]
@QuantumDeveloper what is `validationErrors`? can you privide more code?

**Comment by QuantumDeveloper** [comment]
@punker76 
```
private readonly Dictionary<string, string>
        validationErrors = new Dictionary<string, string>();
```
and How I call the validation method:
```
public FeatureLevel SelectedFeatureLevel
      {
         get { return selectedFeatureLevel; }
         set
         {
            selectedFeatureLevel = value;
            RaisePropertyChanged();
            ValidateFeatureLevel(value);
            if (selectedFeatureLevel != currentFeatureLevel)
            {
               ParametersChanged = true;
            }
            else
            {
               ParametersChanged = false;
            }
         }
      }
```

**Comment by punker76** [comment]
fixed and enabled with ba6cf29eeed3079ebd92c6313a4eddf55aeb12c0
